kurtosis-devnet/pkg/kurtosis/sources/deployer/deployer.go

```diff
@@ -253,13 +254,23 @@ func parseStateFile(r io.Reader) (*DeployerState, error) {
 			continue
 		}
 
-		addresses := mapDeployment(deployment)
+		l1Addresses := mapDeployment(deployment)
 
-		if len(addresses) > 0 {
-			result.Deployments[id] = DeploymentState{
-				Addresses: addresses,
+		// op-deployer currently does not categorize L2 addresses
+		// so we need to map them manually.
+		// TODO: Update op-deployer to sort rollup contracts by category
+		l2Addresses := make(DeploymentAddresses)
+		for _, addressName := range []string{"optimismMintableERC20FactoryProxy"} {
```

Comment:
We can remove this loop? or any other contract name will be added?

Reply:
I think it's nice to establish the pattern for now. Doesn't cost us anything.

Reply:
My thinking is that there may be more L2 contract addresses that we add in the future.